src/clr_bundle_add.c

```diff
@@ -45,6 +45,8 @@ static void print_help(const char *name)
 	printf("Help Options:\n");
 	printf("   -h, --help              Show help options\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
+	printf("   -v, --versionurl=[URL]  RFC-3986 encoded url for version string download\n");
```

Comment:
I was not thinking about format, just content and version urls. But in that sense, the user configuration would have the same issue, or more since with /etc you have a single place to run a post-inst script that converts the format if needed.

Be it in /etc or not, I'd like a best practice to allow us to stick with different versions of the same stream (daily/nightly builds, manually qa-tested and major milestones).